src/compiler/types.ts

```diff
@@ -246,7 +246,7 @@ namespace ts {
         ConditionalExpression,
         TemplateExpression,
         YieldExpression,
-        SpreadElementExpression,
+        SpreadExpression,

```

Comment:
I think the term `SpreadElement` made more sense, since we are spreading an array element. For object spread I think it would be more accurate to call it a `SpreadAssignment` (along with `PropertyAssignment` and `ShorthandPropertyAssignment`).